### PR TITLE
Adds more directional signs. Fixes directions on existing

### PR DESCRIPTION
**Pull request by Peptide90**
## About the PR 
Thought some more signs for my new map would be ideal, especially for helping players get around these new maps where they might not be familiar and don't want to pull up a wiki map. Also noticed the existing directional signs were inconsistent with arrow and directions.

- Fixed the direction of all existing directional signs so placing them is now correct.
- Changed arrow direction for left arrow so all signs now match (see salvage sign in screenshot to see how they were (on the right), now left arrow is on the left for all signs).
- Added directional signs for Hydroponics, Brig, Chapel, Washroom, Dorms, Salvage, Solars, cryo and Gravity Generator.

**Screenshots**

Everything from supply to left is old, everything on the right is new. Supply does in fact face the correct direction now, that was fixed after the screenshot but left it in for comparison.
signs

**Changelog**

:cl:
- add: Added new directional signs for more departments. Look out for these in a future map update.

### CODE REVIEW

File: Resources/Maps/centcomm.yml [hunk 1]
```diff
@@ -25435,7 +25435,7 @@ entities:
     parent: 63
     type: Transform
 - uid: 2526
-  type: SignSecurearea
```
Comment (ElectroJr) [inline Resources/Maps/centcomm.yml:25438]: Seeing as you're already renaming this, you might as well capitalize the "Area"

Comment (Peptide90) [reply Resources/Maps/centcomm.yml:25438]: Makes sense, changed thanks

**Comment by mirrorcult** [comment]
feels like we can easily layer-ize these, like
- base layer (sign base)
- color layer
- arrow direction layer
- area name layer

**Comment by Peptide90** [comment]
> feels like we can easily layer-ize these, like
> 
>     * base layer (sign base)
> 
>     * color layer
> 
>     * arrow direction layer
> 
>     * area name layer

Would this be benificial / cut down on size much? Also since I've corrected the left arrow to the left side would the layering work or would it have to go back to the right?

**Comment by ElectroJr** [comment]
I'm not sure if splitting into re-colourable layers is super useful here, because in this specific instance if you wanted to create a new department sign you'd still have to edit the RSI for the text, and couldn't **just** re-colour the existing sprites. This isn';t the case for stuff like airlocks or door remotes.

And while I have no idea how important it is, I image its easier for clients to draw a single pre-defined layer than 4 separate re-colourable ones, so its probably fine as it is?

**Comment by mirrorcult** [comment]
at least layering the dept marking + arrows would be good. doesnt need to be colored necessarily

**Comment by Peptide90** [comment]
> at least layering the dept marking + arrows would be good. doesnt need to be colored necessarily

So is this the general consensus for this to be merged? I have another few more to add (AI, Head offices or just HEAD).

So would you like them separated into base + text + arrow. And for arrow will it be an issue the left arrow not being on the same side? Also would you need different coloured bases? Say 1 for each department if we don't want loads?

**Comment by ElectroJr** [comment]
The arrow not being on the same side wouldn't be a problem if sprite layer transforms worked properly, but they don't atm. So that might actually be an issue.

> So is this the general consensus for this to be merged? I have another few more to add (AI, Head offices or just HEAD).

I guess get a third opinion, because like I said I don't really think its all that important for these signs to be layered. I don't see people really needing to make use of that in a way where it wouldn't be just as easy to make a whole new sign sprite/state.

**Comment by mirrorcult** [comment]
conflicts, otherwise lgtm

**Review by mirrorcult — APPROVED** [review]

**Comment by metalgearsloth** [comment]
you coming back to this @Peptide90 ?

**Comment by Peptide90** [comment]
> you coming back to this @Peptide90 ?

Will fix conflicts tonight and add the AI and Head / HoP sign.

**Pull request merged**